### ISSUE TITLE
Failed to execute

### ISSUE THREAD

**Issue by yipikai** [Pepitoh/VBad#27]
Hi, 
I was checking your work (which looks amazing btw), but I can't make it work with the example files or some custom ones.
See the following:
![image](https://cloud.githubusercontent.com/assets/28604666/25911754/4b53f362-35b5-11e7-83da-ccc454395789.png)
If you have any idea on what I could do to make this work, it would be much appreciated! 

Thanks
JM

**Comment by Pepitoh** [comment]
Hi, 

Did you check all error message inputs? Please verify that : 
- Your template file is RW
- Check your module name (specific chars?)
- if you check "Trust VBA Project object model" in macro security option on your office suite

Pepitoh.

**Comment by danthegoodman1** [comment]
For me it was enabling the Trust VBA project object model.

**Comment by Pepitoh** [comment]
Did you manage to make it works?

**Comment by johnjohny1990** [comment]
i think you should turn off windows defender

**Comment by Pepitoh** [comment]
Yes, I also see that some AV would block some specific VBA code. But normaly, error messages are not the same (unless your template file is already detected as malicious).

**Comment by johnjohny1990** [comment]
hi
I dont know use this tool
films about this not perfect
please help me

**Comment by X0R1972** [comment]
really guy's is there someone who can give us an example how to use this tool ..seriously i tried everything and nothing works..
please give an example or make a youtube video
